docs/Observability.md

```diff
@@ -44,18 +44,18 @@ The top section of the dashboard shows three metrics using built-in Cosmos DB us
 
 ### Log Analytics - Query-based
 
-The remaining sections of the dashboard focuses on the performance and reliability of the apps (ngsa-cosmos, ngsa-memory, and webv) running on the clusters. The following queries are used to generate the metrics.
+The remaining sections of the dashboard focuses on the performance and reliability of the apps (ngsa-cosmos, ngsa-memory, and loderunner) running on the clusters. The following queries are used to generate the metrics.
 
 > ngsa-cosmos is the app querying against Cosmos DB and ngsa-memory is the app querying against in-memory data.
-> PodType 'webv' is filtered out in some queries as they are focused on the ngsa apps and webv also logs to the same table.
+> PodType 'loderunner' is filtered out in some queries as they are focused on the ngsa apps and loderunner also logs to the same table.
 
```

Comment:
We will need to update these queries to match the query updates for the new logs as well, but I vote for doing those updates in a separate PR.